tests/simulator/test_init_from_restart.py

```
@@ -0,0 +1,84 @@
+import subprocess
+import numpy as np
+import pyphare.pharein as ph
+from pathlib import Path
+
+from pyphare.simulator.simulator import Simulator
+from pyphare.pharesee.hierarchy.patchdata import FieldData, ParticleData
+from pyphare.pharesee.hierarchy.fromh5 import get_all_available_quantities_from_h5
+from pyphare.pharesee.hierarchy.hierarchy import format_timestamp
+from pyphare.pharesee.hierarchy.hierarchy_utils import single_patch_for_LO
+from pyphare.pharesee.hierarchy.hierarchy_utils import hierarchy_compare
+from tests.simulator import SimulatorTest, test_restarts
```

Ruff

`tests.simulator.test_restarts` imported but unused

Remove unused import: `tests.simulator.test_restarts`

(F401)